packages/common/index.ts

```diff
@@ -1115,7 +1076,7 @@ export class UpdateQueueParams {
 }
 
 export class QuestionTypeParams {
-  @IsInt()
+  @IsInt() // when updating a question with new questionTypes, the question type's id is required
```

Comment:
If it's required, we need to remove IsOptional?

Reply:
it's used in other places without id being required (such as when creating questionTypes). I put that comment there since I was analyzing some code and realised that the question type id was required for updating questions.
I mean the real solution is to just have different request types, but that was kinda out of what i wanted to do with this PR